tests/test_legacyrecommender.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from taar.recommenders import LegacyRecommender
+
+FAKE_LEGACY_DATA = {"{test_guid_1}": ["test_guid_2", "test_guid_3"],
```

Comment:
nit: can you reformat this to look like [that](https://github.com/mozilla/taar/blob/921ae67e5dd0d6b26da96a3daa6c167cdfbf4143/tests/test_localerecommender.py#L5)?